src/data/faq.json

```
@@ -884,9 +884,7 @@
                         ]
                     },
                     {
-                        "title": ["Why does the CWA not scan a QR code?",
-                                "  Why does the QR code scanner not respond?"
-                            ],
+                        "title":"Why does the CWA not scan a QR code?, Why does the QR code scanner not respond?",

```

```suggestion
                        "title": "Why does the CWA not scan a QR code? Why does the QR code scanner not respond?",
```
Remove comma, so it is like other titles with multiple sentences.

---

yep, thanks